docs/src/pages/homepage.module.css

```diff
@@ -0,0 +1,275 @@
+/*
+
+Styles for the Nebari home page
+
+*/
+
+/* Create max-width single column for the page's content */
+.sectionContent {
+  align-items: center;
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+.sectionHeading {
+  font-size: 45px;
```

Comment:
The units, sizes, lengths are a bit of a mess across this CSS file: the result of working quickly and copy-pasting some of the styles from the Figma file.

I wasn't sure how much time to spend on this though, so I aimed for "okay" rather than "perfect."

Reply:
ok is good for a first pass - and we can improve this later